tests/run-tests.sh

```diff
@@ -193,6 +201,8 @@ run_test ()
                 STATUS=0
             elif [ -n "${WANT_ABORT}" ] && [ "${STATUS}" -eq "255" ]; then
                 STATUS=0
+            elif [ -n "${WANT_COREDUMP}" ] && [ "${STATUS}" -eq "255" ]; then
+                grep -q "unikernel core file" ${LOGS} && STATUS=0
```

Comment:
could also check that the core file is there

Reply:
We wouldn't know the name of the file since it's core.ukvm.<pid>. Also, if we run ./runtests.sh, multiple times, there will several core files generated eating up the space. We could always delete the core file after each ./runtests but I prefer just naming the core file `core.ukvm`. Thoughts?